packages/vscode-extension/src/utilities/evaluateJsFromCdp.ts

```diff
@@ -0,0 +1,73 @@
+import WebSocket from "ws";
+import { Logger } from "../Logger";
+
+export async function evaluateJsFromCdp(
```

Comment:
this is a weird choice for the method name. We don't evaluate "from" CDP but rather "over" CDP